pkg/skaffold/hooks/host_test.go

```diff
@@ -0,0 +1,67 @@
+// +build !windows
+
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hooks
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	v1 "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest/v1"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+func TestRun(t *testing.T) {
```

Comment:
Since the content between host_test.go and host_win_test.go is virtually midentical, can we not have a single copy and instead embed the GOOS into the struct to help the test decide whether output should be seen or not?